HW_2/task_2.py

```diff
@@ -0,0 +1,20 @@
+"""
+Напишите программу, которая принимает на вход натуральное число N и выдает список факториалов по основаниям от 1 до N
+
+Ввод: значение типа <int>
+Вывод: значение типа <list>
+
+Пример:
+4
+[1, 2, 6, 24]
+"""
+print('Введите число')
+n = int(input())
+f = 1
+if (n % 1 == 0 and n >= 0):
```

Comment:
Тут скобки не нужны.
```suggestion
if n % 1 == 0 and n >= 0:
```